README.md

```diff
@@ -44,6 +44,7 @@ module.exports = {
 |**`include`**|`{RegExp\|Array<RegExp>}`|`undefined`|Files to `include`|
 |**`exclude`**|`{RegExp\|Array<RegExp>}`|`undefined`|Files to `exclude`|
 |**`cache`**|`{Boolean\|String}`|`false`|Enable file caching|
+|**`cacheKeys`**|`{Function(defaultCacheKeys, file) -> {Object}}`|`defaultCacheKeys => defaultCacheKeys`|Allows you to override default cache keys|
```

Comment:
What about the 3rd param?

Reply:
seconded

Reply:
Don't understand :smile:

Reply:
@evilebottnawi sorry. that means "I agree, and have the same comment"

Reply:
@shellscape i don't understand what i do :smile: 3rd param? What is mean?

Reply:
/cc @ooflorent

Reply:
Example below:

```js
[
  new UglifyJsPlugin({
    cache: true,
    cacheKeys: (defaultCacheKeys, file, options) => {
      //                                ^^^^^^^
      defaultCacheKeys.myCacheKey = 'myCacheKeyValue';
      
      return defaultCacheKeys;
    },
  })
]
```

I now understand that my comment was wrong. The signature is correct, the example is not.